_episodes/07-tasks-and-automation.md

```diff
@@ -1,19 +1,114 @@
+
+## What are tasks?
+A task is a series of MarcEdit functions to manipulate data. In the previous lessons, we've run one function at a time. Tasks string those functions together to run at once. Tasks run each function one after another. As such, it is important to think about which tasks follow other tasks. For instance, if you want to add a proxy and the MARC data already has a proxy that you need to delete, your task has to account for that and remove the old proxy before adding a new one.
+
+Tasks are managed through the Task Manager in the MarcEditor. To access the Task Manager, go to Tools and then Manage Tasks in the top level menu. This will open the new tasks window.
+
+In this window, you will see any tasks that you have created previously. You will see the name that you gave those tasks, the path or where those files are saved on your local computer, a shortcut, and a description. At the bottom of this window you will also have two options "Task Actions" and "Manage Existing Tasks".
+
+Task Actions include:
+- New Task: This will create a new task.
+- Clone: This will duplicate an existing task.
+- Delete: This will remove a task.
+- Edit: This will allow you to edit an existing task.
+- Rename: This is how to give a different name to an existing task.
+
+Manage Existing Tasks include:
+- Assign to Group: You can put tasks into different categories or groups.
+- Assign keystrokes: You can assign shotcut keys to tasks.
+- Backup all tasks: Make sure your work is backed up.
+- Delete Task Group: This is to remove an existing group.
+- Export Task File: This will save a task to your local computer.
+- Import Task File: This will bring in a task from your local computer.
+- New Task Group: This will allow you to create a new group.
+- Relink tasks: This will allow you to relink tasks.
+- Remove From Group: This will let you remove a task from a group.
+- Rename Group: This will allow you to rename a group.
+
+The difference between these two actions lists might seem confusing at first. Task actions allow you to work with a particular task in terms of creating and managing one specific task at a time. The Manage Existing Tasks menu allows you to create and manage groups, exporting and importing, and assigning keyboard shortcuts to tasks.
+
+Tasks don't have to be categorized in groups or be linked to keyboard shortcuts. This is up to the users and their preference for how they organize their task workspace.
+
+## Create and a new task
+To create a task in the MarcEditor, Go to Tools in the top level menu and then Manage Tasks. In the Task Actions menu, select "New Task". This opens the task list window where you will see the name you gave your task, a description, tasks, and actions.
```

Comment:
This is different than I experience this workflow. I select "New Task" and a dialogue box comes up with the instruction "Enter Task Name" and a text box. Then you click OK.